splink/database_api.py

```diff
@@ -231,17 +227,13 @@ def register_multiple_tables(
         input_aliases: Optional[List[str]] = None,
         overwrite: bool = False,
     ) -> Dict[str, SplinkDataFrame]:
+        input_tables = self.process_input_tables(input_tables)
 
         tables_as_splink_dataframes = {}
         existing_tables = []
 
         if not input_aliases:
-            # If any of the input_tables are strings, this means they refer
```

Comment:
This doesn't work if the user provides e.g. fake_1000.parquet (a filename rather than a table).  To be consistent with what happens when we register tables in linker.py, we'll just give every table an alias for now

[f"__splink__input_table_{i}" for i, _ in enumerate(input_tables)](https://github.com/moj-analytical-services/splink/blob/da9a901630942b2022b10d6e5b91f1a0f97c447b/splink/linker.py#L436)